File: db4o.net/Db4objects.Db4o.Optional/Db4objects.Db4o/Consistency/SlotDetail.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4objects.Db4o.Internal.Slots;

namespace Db4objects.Db4o.Consistency
{
	public abstract class SlotDetail
	{
		public readonly Slot _slot;

		public SlotDetail(Slot slot)
		{
			this._slot = slot;
		}
	}
}

File: db4oj.optional/core/src/com/db4o/consistency/SlotDetail.java
/* Copyright (C) 2004 - 2010  Versant Inc.  http://www.db4o.com */
package com.db4o.consistency;

import com.db4o.internal.slots.*;

public abstract class SlotDetail {
	public final Slot _slot;

	public SlotDetail(Slot slot) {
		this._slot = slot;
	}
	
}